app/controllers/api/marketing/lists_controller.rb

```diff
@@ -0,0 +1,665 @@
+# 
```

Comment:
But yeah, to your question, I think it's acceptable if the controller is just a one-line shell. I think even better would be for it to separate out the dependencies on the invocation context, so pull the things out of `request` and `params` that you need and pass them to Api:V1::Marketing::Lists.